### ISSUE TITLE
Request compat for: Eidolon

### ISSUE THREAD

**Issue by Wubadubb** [TeamLapen/VampirismIntegrations#42]
I would like to see integrations added for the mod mentioned in the title.  
I confirm it is available for the latest Minecraft version at the time of writing.

## About the mod
Link to the mod page (e.g. on CurseForge): https://www.curseforge.com/minecraft/mc-mods/eidolon

Link to mod source/API:  https://github.com/elucent/eidolon

## What should be integrated exactly
I would like to see Vampires compatible with rituals such as crystalize (undead mobs die and turn into drops) and I would also like to see them affected by the arcane ash. (they, like other hostile undead mobs would not be able to pass through the ash)

## Additional comments
Basically that's all. It's a great mod, sort've like witchery but with its own twists.

**Comment by Cheaterpaul** [comment]
Eidolon is not available for versions past 1.16

This issue can be reopened when the mod updates
